sematic/api/endpoints/tests/test_metrics.py

```diff
@@ -108,18 +108,6 @@ def test_run_state_changed(persisted_run: Run):  # noqa: F811
                 "columns": ["function_path"],
             },
         ),
-        (
```

Comment:
These tests are non-deterministic depending on the platform and timezone settings.